### PR TITLE
rpmostreepayload: do not require network for dvd installation (#1565369)

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: pyanaconda/payload/rpmostreepayload.py [hunk 1]
```diff
@@ -64,6 +64,14 @@ def spaceRequired(self):
         # We don't have this data with OSTree at the moment
         return Size("500 MB")
 
+    @property
+    def needsNetwork(self):
+        """Test ostree repository if it requires network."""
+        if self.data.ostreesetup.url and self.data.ostreesetup.url.startswith("file://"):
```
Comment (cgwalters) [inline pyanaconda/payload/rpmostreepayload.py]: Could just be:
`return not (self.data.ostreesetup.url and self.data.ostreesetup.url.startswith("file://"))`
but LGTM as is too!

Comment (AdamWill) [reply pyanaconda/payload/rpmostreepayload.py]: this is how pylint likes it, IIRC, yes :)

**Comment by AdamWill** [comment]
+1!

File: pyanaconda/payload/rpmostreepayload.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/payload/rpmostreepayload.py]: I decided for if command because somehow it felt more readable to my weary brain, but here you are :)

**Pull request merged**